docs/index.html

```diff
@@ -685,9 +685,9 @@ <h3 slot="header">Popovers</h3>
 
                     <div class="uofi-default-block il-formatted">
                         <div class="uofi uofi-card " style="width: 500px; max-width: 100%;">
-                            <il-clickable-card href="/Design-Resources/Cookie-Banner.html">
-                                <img decoding="async" src="./images/cookie-banner-about.png" alt="" slot="image">
-                                <h3 slot="header">Cookie Banner</h3>
```

Comment:
@fleming7 Is removing the cookie banner from the index page features intentional? My thinking in adding it was that many developers may feel obligated to use it, so I was trying to make it easily discovered.